lib/streamlit/DeltaGenerator.py

```diff
@@ -2048,13 +2051,24 @@ def file_uploader(self, element, label, type=None, encoding="auto", key=None):
             The encoding to use when opening textual files (i.e. non-binary).
             For example: 'utf-8'. If set to 'auto', will try to guess the
             encoding. If None, will assume the file is binary.
+        key : str
+            An optional string to use as the unique key for the widget.
+            If this is omitted, a key will be generated for the widget
+            based on its content. Multiple widgets of the same type may
+            not share the same key.
+        accept_multiple_files : bool
+            If True, the uploader widget will accept multiple files, and the
+            returned value will be a list of all files. Defaults to False.
 
         Returns
         -------
-        BytesIO or StringIO or None
-            The data for the uploaded file. If the file is in a well-known
-            textual format (or if the encoding parameter is set), returns a
-            StringIO. Otherwise BytesIO. If no file is loaded, returns None.
+        BytesIO or StringIO or or list of BytesIO/StringIO or None
+            If no file has been uploaded, returns None. Otherwise, returns
+            the data for the uploaded file(s):
+            - If the file is in a well-known textual format (or if the encoding
+            parameter is set), the file data is a StringIO.
+            - Otherwise the file data is BytesIO.
+            - If multiple_files is True, a list of file datas will be returned.
```

Comment:
Maybe, that's "Italian English"... Technically, `data` is a plural. Can you change to `a list of the data of multiple files` or similar.

Reply:
(I think) you're technically correct, but I disagree that the reworded version is clearer. I think the _actually correct_ grammar here might "a list of files data" - but that is also awkward and would make me confused if I were reading the docs.

Reply:
Just `datas` does not feel right to me.